doc/develop/extending.rst

```diff
@@ -160,121 +158,88 @@ Setting up the development environment on Windows
-    the Odemis virtualenv.
-#.  Install wheel, so we can install binary packages using pip:
-    ``pip install wheel``.
-#.  Install Python future using ``pip install future``
-#.  Install Yaml using ``pip install PyYAML``
-#.  Install 0MQ using ``pip install pyzmq``
-#.  Install the decorator module using ``pip install decorator``
-#.  Install Pillow, a repackaged version of PIL: ``pip install Pillow``
-#.  Install wxPython using ``pip install wxPython==4.0.6``
-#.  Install Delmic's special version of Pyro4:
-    ``pip install git+https://github.com/delmic/Pyro4.git``
-#.  Install Numpy using ``pip install numpy-1.16.5+mkl-cp35-cp35m-win32.whl``,
-    downloaded from http://www.lfd.uci.edu/~gohlke/pythonlibs/#numpy
-#.  Install Libtiff using ``pip install libtiff-0.4.2-cp35-cp35m-win32.whl``, downloaded
-    from http://www.lfd.uci.edu/~gohlke/pythonlibs/#pylibtiff
-#.  Install Scipy using ``pip install scipy-1.2.2-cp35-cp35m-win32.whl``, downloaded from
-    http://www.lfd.uci.edu/~gohlke/pythonlibs/#scipy
-#.  Install OpenCV using ``pip install opencv_python-3.4.7-cp35-cp35m-win32.whl``,
-    downloaded from http://www.lfd.uci.edu/~gohlke/pythonlibs/#opencv
-#.  Install H5py using ``pip install h5py-2.10.0-cp35-cp35-win32.whl``, downloaded
-    from http://www.lfd.uci.edu/~gohlke/pythonlibs/#h5py
-#.  Install Matplotlib using ``pip install matplotlib-2.2.4-cp35-cp35m-win32.whl``,
-    downloaded from http://www.lfd.uci.edu/~gohlke/pythonlibs/#matplotlib
-#.  Install PyCairo using ``pip install pycairo-1.18.1-cp35-cp35m-win32.whl``,
-    downloaded from http://www.lfd.uci.edu/~gohlke/pythonlibs/#pycairo
+Install `Anaconda <https://www.anaconda.com/distribution/>`_ with Python 2.7 32 bits
+(or Python 3.6, but you will have to adjust some of the commands).
 
-Building Cython module(s)
--------------------------
+Open the *Anaconda prompt* and type::
 
-Some parts of Odemis are written with Cython, for optimization reasons. To build these modules on
-MS Windows, first install Visual Studio 2015, which can be found here:
+   cd Documents\odemis
+   conda create -y --name odemisdev python==2.7.16
+   conda activate odemisdev
+   conda config --append channels conda-forge
+   # Edit requirements.txt and remove remove Pyro4
+   conda install --name odemisdev --file requirements.txt
+   conda develop src
+   pip install git+https://github.com/delmic/Pyro4.git
 
-https://www.microsoft.com/en-us/download/details.aspx?id=48159
+Download and install `Microsoft Visual C++ Compiler for Python 2.7 <https://www.microsoft.com/download/details.aspx?id=44266>`_.
+If you use Python 3.6, download and install `Build Tools for Visual Studio 2019 <https://www.visualstudio.com/downloads/#build-tools-for-visual-studio-2019>`_. 
 
-Now copy these files:
-``rc.exe`` and
-``rc.dll``
+You can finally install pylibtiff using::
 
-from ``C:\Program Files (x86)\Windows Kits\8.1\bin\x86``
+   pip install libtiff
```

Comment:
It might not be completely clear that you need to do this in the activated conda env.